src/api/app/helpers/cloud/upload_job_helper.rb

```diff
@@ -0,0 +1,7 @@
+module Cloud
+  module UploadJobHelper
+    def upload_data
```

Comment:
Why this method needs to be in a helper? Is not used in any view...but in a controller. And BTW it relies on a method of the controller.

Reply:
@mdeniz Better?